runtime/cudaq/platform/default/rest_server/RemoteRESTQPU.cpp

```diff
@@ -0,0 +1,365 @@
+/*******************************************************************************
```

Comment:
Do you want this file to instead be named `RemoteSimulatorQPU.cpp`?

Reply:
Indeed, renamed as suggested.